cmd/odf/set/log_level.go

```diff
@@ -37,7 +37,7 @@ func validateLogLevelArgs(_ *cobra.Command, args []string) error {
 	}
 
 	// validate log level
-	match, err := regexp.MatchString("^[1-9][0-9]?$", args[1])
+	match, err := regexp.MatchString("^[1-9][0-9]?(\\/[1-9][0-9]?)?$", args[1])
```

Comment:
Please add a comment that shows examples of valid values.

Reply:
added examples.